slam-ws/src/caffeine_description/launch/caffeine_rviz.launch

```diff
@@ -0,0 +1,11 @@
+<!-- This file is mostly used for messing around w/ rviz and visualizes -->
```

Comment:
This currently does not work, but is it not needed to demonstrate the functionality of the robot